gnd/src/main/java/com/google/android/gnd/persistence/local/room/AreaEntity.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.gnd.persistence.local.room;
+
+import androidx.annotation.NonNull;
+import androidx.room.ColumnInfo;
+import androidx.room.Entity;
+import androidx.room.PrimaryKey;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.LatLngBounds;
+import com.google.android.gnd.model.basemap.Area;
+import com.google.auto.value.AutoValue;
+
+/** Represents a {@link Area} in the local data store. */
+@AutoValue
+@Entity(tableName = "area")
+public abstract class AreaEntity {
+  public static Area toArea(AreaEntity areaEntity) {
+    LatLng northEast = new LatLng(areaEntity.getNorth(), areaEntity.getEast());
+    LatLng southWest = new LatLng(areaEntity.getSouth(), areaEntity.getWest());
+    LatLngBounds bounds = new LatLngBounds(northEast, southWest);
+
+    Area.Builder area =
```

Comment:
Nit: We don't need the intermittent variable `area` here.

Reply:
Also, can we move these static methods after the variables of the entity are declared?

Reply:
Yes! My formatter keeps moving stuff around whenever I run it :/